include/pybind11/pytypes.h

```diff
@@ -1345,11 +1349,20 @@ class weakref : public object {
 class slice : public object {
 public:
     PYBIND11_OBJECT_DEFAULT(slice, object, PySlice_Check)
```

Comment:
Should any of these constructors be explicit?

Reply:
There aren't any here with only one argument? (that is, we don't set default arguments for the new constructor / modified constructor)